### PR TITLE
Cleanup webpack

### PR DESCRIPTION
**Pull request by austinhinderer**

### CODE REVIEW

File: webpack.common.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    new HtmlWebpackPlugin({
+      template: path.join(__dirname, "src", "options.html"),
+      filename: "options.html",
+      chunks: ["options"]
+    }),
+    new HtmlWebpackPlugin({
+      template: path.join(__dirname, "src", "background.html"),
+      filename: "background.html",
+      chunks: ["background"]
+    }),
```
Comment (slipo) [inline webpack.common.js]: I believe we need this added https://github.com/CityOfZion/NeoLink/commit/05daccd704a4290abfd530f5da29fa7b1e2642fc#diff-11e9f7f953edc64ba14b0cc350ae7b9d

**Comment by slipo** [comment]
One problem that seems like a regression, although I'm not sure how it happened, is that clicking on menu links does not make the menu disappear. Really annoying UX so I think we should fix that. Works for me in the development branch.

File: webpack.common.js (hunk 1 above)
Comment (austinhinderer) [reply webpack.common.js]: I actually already have that in my local branch. I'm not sure why it isn't showing up in the diff.

**Comment by slipo** [comment]
For the menu issue what about putting the Menu.Items inside the Links instead of the opposite which we have now. 

Also, here's another issue I'm seeing:

screen shot 2018-01-08 at 7 46 36 am

**Comment by slipo** [comment]
Fixes to the issues brought up in my comments above are in here:
https://github.com/CityOfZion/NeoLink/pull/40

I think if we reimplement this branch's changes on top of that changeset it should go smoothly.

**Pull request closed without merging**